tests/evaluators/test_consistency.py

```diff
@@ -0,0 +1,77 @@
+from agentsim import Simulator
+from agentsim.utils import llm_eval_call, check_guardrail
+from agentsim.evaluation import evaluator
+
+config = """
+---
+evaluators:
+    consistency_score_llm:
+        model_settings:
+            model: openai/gpt-4o
+            api_key: OPENAI_API_KEY
+            system_prompt: >
+                [Instruction]
+                Please act as an impartial judge and evaluate the quality of the output to the input displayed below. Your evaluation should be based on the mentioned criteria. Begin your evaluation by providing a short explanation on how the output performs relative to the input. Be as objective as possible. After providing your explanation, you must rate the response on a scale of 1 to 5 by strictly following this JSON format: '{"rating": rating}, for example: '{"rating": 2}.
+                [Criteria]
+
+                Rate the consistency of information and stance across multiple messages, with 5 being most consistent and 1 being least consistent.
```

Comment:
Add more qualitative description of the criteria and explain what a score of 1 and 5 imply.